packages/react-core/src/components/Alert/AlertBody.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+const propTypes = {
+  title: PropTypes.string,
+  children: PropTypes.node,
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  title: null,
+  children: '',
+  className: ''
+};
+
+const RenderTitle = title => (
+  <h4 className={css(styles.alertTitle)}>{title}</h4>
+);
+
+const AlertBody = ({ title, className, children, ...props }) => (
+  <div {...props} className={css(styles.alertBody, className)}>
+    {title && RenderTitle(title)}
```

Comment:
no need for `RenderTitle` here.  Just inline it.

Reply:
I was thinking about seperate component in same file or render function and went with some kind of hybrid. I'll probably just inline it.